instrumentation/database/sql/splunksql/test/splunksql_test.go

```diff
@@ -18,9 +18,11 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/suite"
+	"go.uber.org/goleak"
 )
 
 func TestSplunkSQLFullImplementation(t *testing.T) {
+	defer goleak.VerifyNone(t)
```

Comment:
Can you let me know if the test is failing without the `defer db.Close()` in `instrumentation/database/sql/splunksql/sql.go`? 😉